src/components/ui/carousel.tsx

```diff
@@ -160,7 +160,7 @@ const CarouselContent = React.forwardRef<
         ref={ref}
         className={cn(
           'flex',
-          orientation === 'horizontal' ? '-ml-4' : '-mt-4 flex-col',
+          orientation === 'horizontal' ? '' : '-mt-4 flex-col',
```

Comment:
해당 조건일 경우에만 css 속성 적용되도록 해도 좋을 것 같아요~!
```suggestion
          orientation === 'horizontal' && '-mt-4 flex-col',
```

Reply:
83775fd4343f0c69fbaf75ca8fab56935e16eef3 제안 감사합니당